src/sonic-bmpcfgd/tests/common/mock_configdb.py

```diff
@@ -0,0 +1,162 @@
+class MockConfigDb(object):
```

Comment:
>MockConfigDb [](http://example.com/codeflow?start=6&length=12)

If there is coverage data in PR checker, could you collect local test coverage data and attach them so we can check which lines are not covered.

Reply:
Still applicable.

Reply:
Still applicable

Reply:
Refer https://dev.azure.com/mssonic/build/_build/results?buildId=553417&view=codecoverage-tab for coverage result. The uncovered function is callback from swss-common and main function.

Reply:
Latest coverage refer https://dev.azure.com/mssonic/build/_build/results?buildId=558875&view=codecoverage-tab, and below are comparison with hostcfgd, pasted screenshot since official link has displaying issue (https://dev.azure.com/mssonic/build/_build/results?buildId=558402&view=codecoverage-tab) 

![image](https://github.com/sonic-net/sonic-buildimage/assets/97083744/cb068b3f-536b-4894-add6-965b41eb339a)